environment_setup/setup_software.sh

```diff
@@ -132,6 +132,21 @@ if ! sudo /opt/tbotspython/bin/pip3 install pyqt5  ; then
     exit 1
 fi
 
+# TODO (#2515) move iterfzf and fuzzywuzzy to requirements.txt
+if ! sudo /opt/tbotspython/bin/pip3 install iterfzf  ; then
```

Comment:
also install python-Levenshtein for speed

Reply:
ah yes, I installed it locally but forgot to add it to the setup script